scene/gui/text_edit.cpp

```diff
@@ -4314,12 +4314,12 @@ Point2i TextEdit::get_line_column_at_pos(const Point2i &p_pos, bool p_allow_out_
 	return Point2i(col, row);
 }
 
-Point2i TextEdit::get_pos_at_line_column(int p_line, int p_column) const {
-	Rect2i rect = get_rect_at_line_column(p_line, p_column);
+Point2 TextEdit::get_pos_at_line_column(int p_line, int p_column) const {
+	Rect2 rect = get_rect_at_line_column(p_line, p_column);
 	return rect.position.x == -1 ? rect.position : rect.position + Vector2i(0, get_line_height());
 }
 
-Rect2i TextEdit::get_rect_at_line_column(int p_line, int p_column) const {
+Rect2 TextEdit::get_rect_at_line_column(int p_line, int p_column) const {
 	ERR_FAIL_INDEX_V(p_line, text.size(), Rect2i(-1, -1, 0, 0));
```

Comment:
You're still returning `Rect2i` in a lot of places here

Reply:
I pointed to one of the examples of it not having been updated, you missed this one :)